Microsoft.Toolkit.Uwp.UI.Controls.Core/Microsoft.Toolkit.Uwp.UI.Controls.Core.csproj

```diff
@@ -12,12 +12,34 @@
         - ImageEx: Images are downloaded asynchronously showing a load indicator and can be stored in a local cache.
         - InAppNotification: Show local notifications in your application.
         - Loading: Helps to show content with animation to the user while the app is doing some calculation.
+        - Menu: TODO
         - RadialProgressBar: Displays progress as a circle getting filled.
         - RotatorTile: Rotates through a set of items one-by-one like a live-tile.
+        - TabbedCommandBar: TODO
```

Comment:
Needs description.

Reply:
@RosarioPulella can you update the comment on the TabbedCommandBar itself to say:

> A top-level navigation paradigm for presenting a command bar that organizes the features of an application into a series of tabs at the top of the application window. It replaces both the traditional menu bar and toolbars.

I'd like to avoid having 'ribbon' has a descriptor still, and I just noticed that from our API docs. (double-check that we don't have it elsewhere?)

Then for this one we should shorten it (as we have the 4000 character limit) to: "A command bar that organizes features of an application into a series of tabs."

Reply:
So I updated it to, is that cool?
> A command bar that organizes features of an application into a series of tabs